adr/0008-rest-api-versioning-strategy.md

```diff
@@ -0,0 +1,38 @@
+# 8. Rest API versioning strategy
+
+Date: 2023-02-08
+
+## Status
+
+Accepted
+
+## Context
+
+With the growing complexity of our services, we need to stabilize and version our external rest API.
+This guideline will affect all the present and future rest API, we need a standard, easy and maintainable way of doing that stuff.
+This decision affects only the purely technical side, involving the HTTP rest API, without defining when an API should be upgraded.
+That topic will be covered in another part of the documentation
+
+We want a clear API versioning strategy for the API consumer and also a way to express unambiguously the "latest" version of our API.
+
+## Decision
+
+We will implement a URL-based API versioning.
+That means that our routes will be prefixed with `/vX`, where "X" is a progressive number that indicates the current version of the consumed API.
+This means that in our projects, we need to namespace the handlers, requests and responses of our http routes.
+For example in an elixir project, we want to namespace the controllers, the views and the OpenApi schemas with the version, using a module called VX (V1, V2 etc..)
+
+This means that a consumer could choose also the right Schemas for the OpenApi spec consuming, generating a client according to a specific version.
```

Comment:
I would remote this `This means` and put this sentence after the previous one splitted by a dot